src/Tablebot/Plugins/Roll/Plugin.hs

```diff
@@ -181,7 +231,7 @@ statsCommand = Command "stats" statsCommandParser []
               liftDiscord $
                 void $
                   restCall
-                    ( CreateMessageDetailed (messageChannel m) (MessageDetailedOpts (msg range') False Nothing (Just (T.unwords (snd <$> range') <> ".png", toStrict image)) Nothing Nothing)
+                    ( CreateMessageDetailed (messageChannelId m) (MessageDetailedOpts (msg range') False Nothing (Just (T.unwords (snd <$> range') <> ".png", toStrict image)) Nothing Nothing Nothing Nothing)
```

Comment:
Could there be a helper function that avoids all of the `Nothing` here, or is there no way to narrow it down? (Fine if not, just seems like a whole lot of Nothing.)